src/es-module-shims.js

```diff
@@ -329,6 +329,14 @@ new MutationObserver(mutations => {
         processScript(node, !firstTopLevelProcess);
       else if (node.tagName === 'LINK' && node.rel === 'modulepreload')
         processPreload(node);
+      else if (node.querySelectorAll) {
+        for (const script of node.querySelectorAll('script[type="module-shim"],script[type="importmap-shim"],script[type="module"],script[type="importmap"]')) {
```

Comment:
We could possibly simplify this like the above to just:

```suggestion
        for (const script of node.querySelectorAll('script')) {
```

and then let the function handle the rest.

Reply:
I used the same selector as in https://github.com/guybedford/es-module-shims/blob/main/src/es-module-shims.js#L320 Another option could be to store the selector in a `const` and reuse it.

Feel free to adapt the diff how you prefer it. I don't have a strong opinion either way :)